controllers/articles.controller.js

```diff
@@ -1,17 +1,33 @@
-const {getArticleById, getArticles} = require('../models/articles.models')
+const {
+  getArticleById,
+  getArticles,
+  patchVote,
+} = require("../models/articles.models");
 
 exports.findArticleById = (req, res, next) => {
-    const {article_id} = req.params
-    getArticleById(article_id)
+  const { article_id } = req.params;
+  getArticleById(article_id)
     .then((article) => {
-        res.status(200).send(article)
+      res.status(200).send(article);
     })
-    .catch(next)
-}
+    .catch(next);
+};
 
 exports.findArticles = (req, res, next) => {
-    getArticles()
-    .then((articles) => {
-        res.status(200).send({articles: articles})
+  getArticles().then((articles) => {
+    res.status(200).send({ articles: articles });
+  });
+};
+
+exports.updateVotes = (req, res, next) => {
+  const { article_id } = req.params;
+  getArticleById(article_id)
+    .then(({ article_id }) => {
+      const { inc_votes } = req.body;
+      patchVote(article_id, inc_votes)
+      .then((article) => {
+        res.status(200).send({article: article})
```

Comment:
I don't think it should be necessary to do a `getArticleById` here to check ahead to see if the article exists. Because it's possible to use `RETURNING *` on your `UPDATE` query and you could check the return values of `rows` there to see if it is empty